auth_ldap_self_signed/__init__.py

```diff
@@ -1,5 +1,5 @@
 # -*- coding: utf-8 -*-
-# © 2015 Elico corp (www.elico-corp.com)
```

Comment:
both 2015-2017 (everywhere)
In all PR, please check: keep the original date and add the new one.